mysite/mysite/templates/partial/grid-display.html

```diff
@@ -1,20 +1,42 @@
 {% load static %}
-<table id="movieList">
-    {% for row in titles %}
-        <tr>
-        {% for t in row %}
-            <td class="movieList-item">
-                <a href="/titles/{{ t.title_id }}">
-                    {% if t.poster %}
-                        <img src="{{ t.poster }}" alt="{{ t.title }}">
-                    {% else %}
-                        <img src="{% static 'generic-poster.jpg' %}">
-                    {% endif %}
+<script type="text/javascript" src="{% static 'grid-display.js' %}"></script>
+<div id="grid-container">
+    {% if not search %}
+        <div id="page-selector">
```

Comment:
Nice feature.  I think it would look better if it was at the bottom of the grid.